radar-upload-backend/src/main/java/org/radarbase/upload/api/ApiDeclarations.kt

```diff
@@ -30,8 +30,8 @@ data class ContentsDTO(
         var url: String? = null,
         var text: String? = null,
         var contentType: String,
-        var createdDate: Instant,
-        var size: Long,
+        var createdDate: Instant? = null,
```

Comment:
why do we allow createdDate to be `null`?

Reply:
So that the client can upload a record directly in a single request, without going through the POST record, PUT file, POST metadata ready. In this case, the createdDate is set by the backend, not by the user. So for the user it is null.